pages/devPeerReviewWorkflow.md

```diff
@@ -46,7 +52,7 @@ Merges into the `dev` branch need only one approval to merge but give all active
 
 The dev who proposed the PR presses the merge button on a PR once all the above passes & it is approved. That dev is responsible for merging in a way that does not "break the build" or cause unplanned outages.
 
-Press the "Delete Branch" on Github after merging. This keeps the repo tidy.
+Press the "Delete Branch" on Github after merging. This keeps the repo tidy. Should now be automated by the **always-be-closing** bot.
```

Comment:
Github now has a setting for the repo. It will automatically delete the branch (if we turn this feature on)